testnet/council-period-scoring/forum-score.md

```diff
@@ -0,0 +1,145 @@
+---
+description: Metrics used to compute score for the forum working group.
+---
+
+# Forum Score
+
+## Overview
+
+The forum working group activities relevant to scoring will, generally, fall into one of the following categories
+
+* Maintain a reasonable set of categories, allowing users to easily navigate the forum
+* Moderate the forum
+  * ideally, this means moving threads that were placed inappropriately, but also
+  * includes censoring anything that violates the [ToS](todo)
```

Comment:
Could this just link to the concent policy? https://github.com/Joystream/handbook/blob/master/system/content-directory/content-policy.md

It would seem to be quite similar?

Reply:
Good point!